package.json

```diff
@@ -21,7 +21,6 @@
     "@types/react": "^18",
     "@types/react-dom": "^18",
     "@typescript-eslint/eslint-plugin": "^6.21.0",
-    "@typescript-eslint/parser": "^7.2.0",
```

Comment:
It mustt be notated.
By the way, It was not cause of ESLint's no response. ESLint works well.